src/java.prefs/share/classes/java/util/prefs/Preferences.java

```diff
@@ -30,20 +30,10 @@
 import java.io.InputStream;
 import java.io.IOException;
 import java.io.OutputStream;
-import java.security.AccessController;
-import java.security.Permission;
-import java.security.PrivilegedAction;
 import java.util.Iterator;
 import java.util.ServiceLoader;
 import java.util.ServiceConfigurationError;
 
-// These imports needed only as a workaround for a JavaDoc bug
```

Comment:
This comment was added quite a while ago. But FWIW, a specdiff comparison shows no JavaDoc differences.